### ISSUE TITLE
Export aws_session_token from --generate

### ISSUE THREAD

**Issue by abrobston** [linaro-its/aws2-wrap#27]
Using `aws2-wrap` version 1.1.2, If I generate a shared credentials file using `aws2-wrap --generate`, the resulting credentials file is missing the `aws_session_token` value.  As a result, anything requiring a session token that uses the credentials file will fail.

For example:

```
~ % export BASE="$HOME/.bugreport"
~ % mkdir $BASE
~ % export AWS_CONFIG_FILE="$BASE/config"
~ % export AWS_SHARED_CREDENTIALS_FILE="$BASE/credentials"
~ % aws --version
aws-cli/2.1.11 Python/3.9.1 Darwin/19.6.0 source/x86_64 prompt/off
~ % export AWS_PROFILE=bugtest
~ % export AWS_REGION=us-east-1
~ % echo "[profile bugtest]" >> $AWS_CONFIG_FILE
~ % aws configure --profile bugtest
AWS Access Key ID [None]:
AWS Secret Access Key [None]:
Default region name [None]:
Default output format [None]:
~ % aws configure sso --profile bugtest
SSO start URL [None]: https://**************.awsapps.com/start
SSO Region [None]: us-east-1
Attempting to automatically open the SSO authorization page in your default browser.
If the browser does not open or you wish to use a different device to authorize this request, open the following URL:

https://device.sso.us-east-1.amazonaws.com/

Then enter the code:

****-****
There are * AWS accounts available to you.
Using the account ID ###########
There are # roles available to you.
Using the role name "********"
CLI default client Region [None]: us-east-1
CLI default output format [None]:

To use this profile, specify the profile name using --profile, as shown:

aws s3 ls --profile bugtest
1m 13s ~ % cat "$AWS_CONFIG_FILE"
[profile bugtest]
sso_start_url = https://**********.awsapps.com/start
sso_region = us-east-1
sso_account_id = ############
sso_role_name = *******
region = us-east-1
~ % aws sso login
Attempting to automatically open the SSO authorization page in your default browser.
If the browser does not open or you wish to use a different device to authorize this request, open the following URL:

https://device.sso.us-east-1.amazonaws.com/

Then enter the code:

****-****
Successully logged into Start URL: https://************.awsapps.com/start

~ % aws s3 ls --page-size 1

<long list of S3 buckets>

~ % mkdir bug-test-app
~ % cd bug-test-app
~/bug-test-app % npm init -y
Wrote to /Users/******/bug-test-app/package.json:

{
  "name": "bug-test-app",
  "version": "1.0.0",
  "description": "",
  "main": "index.js",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "author": "",
  "license": "ISC"
}


~/bug-test-app % npm install "@aws-sdk/client-s3"

...
+ @aws-sdk/client-s3@3.0.0
added 148 packages from 16 contributors and audited 148 packages in 9.989s
...
11s ~/bug-test-app % ls
./  ../  node_modules/  package-lock.json  package.json
~/bug-test-app % cat <<EOF >index.js
const s3 = require("@aws-sdk/client-s3");

const client = new s3.S3({});

(async () => {
    const result = await client.listBuckets({});
    console.log(\`Bucket count is \${result.Buckets.length}\`);
})();
EOF
~/bug-test-app % node -v
v14.15.1
~/bug-test-app % node index.js
(node:99979) UnhandledPromiseRejectionWarning: Error: Profile bugtest could not be found or parsed in shared credentials file.
    at resolveProfileData (/Users/*****/bug-test-app/node_modules/@aws-sdk/credential-provider-ini/dist/cjs/index.js:80:11)
    at /Users/*****/bug-test-app/node_modules/@aws-sdk/credential-provider-ini/dist/cjs/index.js:29:59
    at async SignatureV4.credentialProvider (/Users/*****/bug-test-app/node_modules/@aws-sdk/property-provider/dist/cjs/memoize.js:26:26)
    at async SignatureV4.signRequest (/Users/*****/bug-test-app/node_modules/@aws-sdk/signature-v4/dist/cjs/SignatureV4.js:84:29)
    at async /Users/*****/bug-test-app/node_modules/@aws-sdk/middleware-signing/dist/cjs/middleware.js:14:22
    at async StandardRetryStrategy.retry (/Users/*****/bug-test-app/node_modules/@aws-sdk/middleware-retry/dist/cjs/defaultStrategy.js:55:46)
    at async /Users/*****/bug-test-app/node_modules/@aws-sdk/middleware-logger/dist/cjs/loggerMiddleware.js:7:22
    at async /Users/*****/bug-test-app/index.js:6:20
(Use `node --trace-warnings ...` to show where the warning was created)
...
~/bug-test-app % aws2-wrap --generate --profile $AWS_PROFILE --credentialsfile $AWS_SHARED_CREDENTIALS_FILE --configfile $AWS_CONFIG_FILE --outprofile bugtest-wrap
Writing Credentials to /Users/*****/.bugreport/credentials
The credentials will expire at 2020-12-21T19:46:06+00:00
Writing Configuration to /Users/*****/.bugreport/config
~/bug-test-app % cat "$AWS_CONFIG_FILE"
[profile bugtest]
sso_start_url = https://*******.awsapps.com/start
sso_region = us-east-1
sso_account_id = ############
sso_role_name = *****
region = us-east-1

[bugtest-wrap]
region = us-east-1
(130) 5s ~/bug-test-app % cat "$AWS_SHARED_CREDENTIALS_FILE"

[bugtest-wrap]
aws_access_key_id = ASI***************XY
aws_secret_access_key = On************************************ty
~/bug-test-app % export AWS_PROFILE=bugtest-wrap
~/bug-test-app % node index.js
(node:834) UnhandledPromiseRejectionWarning: InvalidAccessKeyId: The AWS Access Key Id you provided does not exist in our records.
    at deserializeAws_restXmlListBucketsCommandError (/Users/*****/bug-test-app/node_modules/@aws-sdk/client-s3/dist/cjs/protocols/Aws_restXml.js:6841:41)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async /Users/*****/bug-test-app/node_modules/@aws-sdk/middleware-serde/dist/cjs/deserializerMiddleware.js:7:20
    at async /Users/*****/bug-test-app/node_modules/@aws-sdk/middleware-signing/dist/cjs/middleware.js:12:24
    at async StandardRetryStrategy.retry (/Users/*****/bug-test-app/node_modules/@aws-sdk/middleware-retry/dist/cjs/defaultStrategy.js:55:46)
    at async /Users/*****/bug-test-app/node_modules/@aws-sdk/middleware-logger/dist/cjs/loggerMiddleware.js:7:22
    at async /Users/*****/bug-test-app/index.js:6:20
(Use `node --trace-warnings ...` to show where the warning was created)
...
~/bug-test-app % echo '[bugtest-wrap]' > "$AWS_SHARED_CREDENTIALS_FILE"
~/bug-test-app % aws2-wrap --export --profile bugtest | grep -Fv AWS_DEFAULT_REGION | sed -e 's/"//g' -e 's/=/ = /' -e 's/^export //' | awk -F ' = ' '{print tolower($1)" = "$2}' >> "$AWS_SHARED_CREDENTIALS_FILE"
~/bug-test-app % cat "$AWS_SHARED_CREDENTIALS_FILE"
[bugtest-wrap]
aws_access_key_id = AS****************CK
aws_secret_access_key = wV************************************ov
aws_session_token = IQ*****************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************222I=
~/bug-test-app % node index.js
Bucket count is ##
```

Once `aws_session_token` is set correctly in the shared credentials file, everything seems to work.

**Comment by pcolmer** [comment]
Thank you, @abrobston, for such a comprehensive recipe to reproduce the problem.

**Comment by pcolmer** [comment]
Hi @abrobston 

The bug has been fixed in version 1.1.3.

Many thanks again for the comprehensive recipe.
